ext/data/recommended-settings.json

```diff
@@ -203,6 +305,54 @@
                 "value": "word"
             },
             "description": "Lookup whole words in the dictionary."
+        },
+        {
+            "modification": {
+                "action": "set",
+                "path": "parsing.enableScanningParser",
+                "value": false
+            },
+            "description": "Turn off Yomitan's internal parser for languages with spaces."
+        }
+    ],
+    "ko": [
+        {
+            "modification": {
+                "action": "set",
+                "path": "parsing.enableScanningParser",
+                "value": false
+            },
+            "description": "Turn off Yomitan's internal parser for languages with spaces."
+        }
+    ],
```

Comment:
Is this a mistake?

Reply:
Nvm korean characters look kindof like the space is part of the character or font and tricked me.